### ISSUE TITLE
Unsafe legacy lifecycles warring for Pickers

### ISSUE THREAD

**Issue by ismarosmanovic** [GeekyAnts/NativeBase#2881]
## I have gone through these following points
- []   Check latest documentation: https://docs.nativebase.io/
- []   Check for existing open/closed issues for a possible duplicate before creating a new issue: https://github.com/GeekyAnts/NativeBase/issues
- []   Use the latest NativeBase release: https://github.com/GeekyAnts/NativeBase/releases
- []   Check examples from NativeBase KitchenSink https://github.com/GeekyAnts/NativeBase-KitchenSink
- []   For discussion purpose make use of NativeBase Slack: http://slack.nativebase.io/
- []   For queries related to theme, check [Theme Variables](https://docs.nativebase.io/docs/ThemeVariables.html) from Docs and live NativeBase Theme Editor http://nativebase.io/customizer/

## Issue Description
Adding picker component to any page throws following warning:
******************************************
Warning: Unsafe legacy lifecycles will not be called for components using new component APIs.

%s uses %s but also contains the following legacy lifecycles:%s%s%s

The above lifecycles should be removed. Learn more about this warning here:
https://fb.me/react-async-component-lifecycle-hooks, Styled(PickerNB), getDerivedStateFromProps(), , 
  UNSAFE_componentWillReceiveProps, 
******************************************
### node, npm, react-native, react and native-base version, expo version if used, xcode version
native-base: 2.13.5,
expo: 34.0.4

,

### Steps to reproduce
Just add Picker to the page. 

### Is the bug present in both iOS and Android or in any one of them?
Only on IOS

<img width="374" alt="Screenshot 2019-08-17 at 23 06 07" src="https://user-images.githubusercontent.com/45101366/63216810-b2e78e00-c143-11e9-89f9-67809b7a35ae.png">

**Comment by yitelee** [comment]
same here, IOS only

**Comment by mccordgh** [comment]
same issue as well:
native-base: 2.13.5
expo: 32.0.0

The picker still seems to work as normal, I just get the warning every time I navigate to that screen.

**Comment by halstjas** [comment]
I have also noticed this with the version of react-native-keyboard-aware-scroll-view v0.8 which is being bundled with the latest version of native-base which is using react deprecated lifecycle method componentWillReceiveProps can you please update to use library to latest v0.9.1 which resolves this issue.

**Comment by GitHubGreg** [comment]
Same here (same as OP).

**Comment by dsusviela** [comment]
Same

**Comment by siliconavengers** [comment]
same. We hope someone will submit a PR for this warning.

**Comment by sankhadeeproy007** [comment]
Can we try it with the latest version 2.13.7 and see if the issue still persists?

**Comment by mccordgh** [comment]
@sankhadeeproy007 Still having the issue with 2.13.7

**package-lock.json:**
![Screen Shot 2019-09-09 at 9 53 32 AM](https://user-images.githubusercontent.com/11222008/64550444-cb346e00-d2e7-11e9-9d4c-f7032f562c16.png)

**in-app screenshot:**
![IMG_5370A91DBB64-1](https://user-images.githubusercontent.com/11222008/64550488-e1dac500-d2e7-11e9-99f3-758cd3ebf187.jpeg)

**warning from terminal:**
```
[09:52:54] Warning: Unsafe legacy lifecycles will not be called for components using new component APIs.

%s uses %s but also contains the following legacy lifecycles:%s%s%s

The above lifecycles should be removed. Learn more about this warning here:
https://fb.me/react-async-component-lifecycle-hooks, Styled(PickerNB), getDerivedStateFromProps(), ,
  UNSAFE_componentWillReceiveProps,
```

**Comment by Brayden967** [comment]
Exact same problem for me, any updates?

**Comment by sankhadeeproy007** [comment]
Fixed in 2.13.8
